static/styles/zulip.css

```diff
@@ -1388,10 +1388,127 @@ blockquote p {
     font-weight: bold;
 }
 
+.alert-box {
```

Comment:
Probably. I don't know whether I should first restyle other alerts (other than the ones that were historically in the right sidebar).